website/docs/r/event_orchestration_service.html.markdown

```diff
@@ -0,0 +1,202 @@
+      conditions {
+        expression = "event.custom_details.hostname matches part 'canary'"
+      }
+      # create webhook action with parameters and headers
+      actions {
+        automation_actions {
+          name = "Canary Slack Notification"
+          url = "https://our-slack-listerner.test/canary-notification"
+          auto_send = true
+          parameters {
+            key = "channel"
+            value = "#my-team-channel"
+          }
+          parameters {
+            key = "message"
+            value = "something is wrong with the canary deployment"
+          }
+          headers {
+            key = "X-Notification-Source"
+            value = "PagerDuty Incident Webhook"
+          }
+        }
+      }
+    }
+    rules {
+      label = "Never bother the on-call for info-level events outside of work hours"
+      conditions {
+        expression = "event.severity matches 'info' and not (now in Mon,Tue,Wed,Thu,Fri 09:00:00 to 17:00:00 America/Los_Angeles)"
+      }
+      actions {
+        suppress = true
+      }
+    }
+  }
+  catch_all {
+    actions { }
+  }
+}
+```
+## Argument Reference
+
+The following arguments are supported:
+
+* `service` - (Required) ID of the Service to which this Service Orchestration belongs to.
+* `sets` - (Required) A Service Orchestration must contain at least a "start" set, but can contain any number of additional sets that are routed to by other rules to form a directional graph.
+* `catch_all` - (Required) When none of the rules match an event, the event will be routed according to the catch_all settings.
+
+### Sets (`sets`) supports the following:
+* `id` - (Required) The ID of this set of rules. Rules in other sets can route events into this set using the rule's `route_to` property.
+* `rules` - (Optional) The service orchestration evaluates Events against these Rules, one at a time, and routes each Event based on the first rule that matches. If no rules are provided as part of Terraform configuration, the API returns empty list of rules.
+
+### Rules (`rules`) supports the following:
+* `label` - (Optional) A description of this rule's purpose.
+* `conditions` - (Optional) Each of these conditions is evaluated to check if an event matches this rule. The rule is considered a match if any of these conditions match. If none are provided, the event will `always` match against the rule.
+* `actions` - (Required) Actions that will be taken to change the resulting alert and incident, when an event matches this rule.
+* `disabled` - (Optional) Indicates whether the rule is disabled and would therefore not be evaluated.
+
+### Conditions (`conditions`) supports the following:
+* `expression`- (Required) A [PCL condition](https://developer.pagerduty.com/docs/ZG9jOjM1NTE0MDc0-pcl-overview) string.
+
+### Actions (`actions`) supports the following:
+* `route_to` - (Required) The ID of the target Service for the resulting alert.
```

Comment:
**Blocker:** `route_to` is optional for service orchs & this text seems to be for the router. Maybe this wording taken from the API docs instead?

```suggestion
* `route_to` - (Optional) The ID of a Set from this Service Orchestration whose rules you also want to use with event that match this rule.
```